package/MDAnalysis/analysis/base.py

```diff
@@ -27,6 +27,8 @@
 
 import numpy as np
 import logging
+from MDAnalysis.lib.log import ProgressMeter
```

Comment:
I'm using atom and messing around with code in jupyter. I used to use vim
but atom does a great job with git integration.

On Tue, May 24, 2016 at 8:58 AM kain88-de <notifications@github.com> wrote:

> In package/MDAnalysis/analysis/base.py
> <https://github.com/MDAnalysis/mdanalysis/pull/856#discussion_r64420578>:
>
> > @@ -27,6 +27,8 @@
> >
> >  import numpy as np
> >  import logging
> > +from MDAnalysis.lib.log import ProgressMeter
>
> Your editor should tell you about that. I use elpy with emacs but if you
> prefer IDE's I hear pycharm is good and has a PEP8 checker included.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/MDAnalysis/mdanalysis/pull/856/files/c41cf9540291e37191f253bfde996a7e0a6f3767#r64420578>
>
-- 
Have a wonderful day,
John J. Detlefs